.gitignore

```diff
@@ -12,3 +12,7 @@ export_presets.cfg
 # Mono-specific ignores
 .mono/
 data_*/
+
+# Replay-specific ignores
```

Comment:
Following the idea that functions should expect the path to the replay file, it becomes responsibility of the project itself to deal with version control ignores.

Reply:
Totally agree. Also, after sleeping on it, I think:

1) A replay folder in the project itself is a redundant and overly-complex idea. Even if someone wants to separate where replays are stored based on parameters such as debug and/or in-editor builds, other scripts can decide where how files are accessed.

2) On a related note to letting someone choose how they want to structure their file setup, I think another change I should make to how the replay class interacts with the filesystem is to get rid of save path constants, and just let per-project functions take care of choosing where to save and access replay files.

Summary: Probably best to simply not update the .gitignore because its specific ignores will be deprecated soon and storing replay files in a git project is a bad habit.

Reply:
Just read the remaining comments you left... We agree the file locations shouldn't be hard coded, haha! Sorry I mentioned that as if you hadn't already...